Library/Homebrew/extend/os/cmd/cask.rb

```diff
@@ -0,0 +1,5 @@
+if OS.mac?
+  require "extend/os/mac/cmd/cask"
+else
+  odie "Homebrew Cask is supported on macOS only."
```

Comment:
I'd suggest…
```
The cask command requires macOS.
```

Reply:
maybe I'm not correct, but it seems weird for a command to _require_ an operating system, no?
A command/program can require another program/command but it is either supported or not supported by the operating system.

Reply:
`Library/Homebrew/requirements/macos_requirement.rb` says `macOS is required.`

Reply:
I think that requiring an operating system is fine. e.g. "iTunes requires either macOS or Windows."

Reply:
> `Library/Homebrew/requirements/macos_requirement.rb` says `macOS is required`.

```
$ git blame Library/Homebrew/requirements/macos_requirement.rb 
8f728d3a (Shaun Jackman 2017-02-25 10:17:25 -0800 1) class MacOSRequirement < Requirement
8f728d3a (Shaun Jackman 2017-02-25 10:17:25 -0800 2)   fatal true
8f728d3a (Shaun Jackman 2017-02-25 10:17:25 -0800 3) 
8f728d3a (Shaun Jackman 2017-02-25 10:17:25 -0800 4)   satisfy(build_env: false) { OS.mac? }
8f728d3a (Shaun Jackman 2017-02-25 10:17:25 -0800 5) 
8f728d3a (Shaun Jackman 2017-02-25 10:17:25 -0800 6)   def message
8f728d3a (Shaun Jackman 2017-02-25 10:17:25 -0800 7)     "macOS is required."
8f728d3a (Shaun Jackman 2017-02-25 10:17:25 -0800 8)   end
8f728d3a (Shaun Jackman 2017-02-25 10:17:25 -0800 9) end
```

:)


There is no `Library/Homebrew/requirements/macos_requirement.rb` in Homebrew

Reply:
Ah, hasn't been merged yet. See https://github.com/Homebrew/brew/pull/3270

Reply:
I suggest moving this file to `os/cmd/cask.rb`